gui/qt/main_window.py

```diff
@@ -2248,7 +2248,7 @@ def do_encrypt(self, message_e, pubkey_e, encrypted_e):
             encrypted_e.setText(encrypted.decode('ascii'))
         except BaseException as e:
             traceback.print_exc(file=sys.stdout)
-            self.show_warning(str(e))
+            self.show_warning("Invalid Public key")
```

Comment:
Let's allow translators for this string to be translated:
`self.show_warning(_('Invalid Public key'))`